tests/framework/installer/ceph_manifests.go

```diff
@@ -243,13 +243,6 @@ spec:
 `
 	}
 
-	if m.settings.ConnectionsEncrypted {
-		clusterSpec += `
-  csi:
-    cephfs:
-      kernelMountOptions: ms_mode=secure
```

Comment:
@travisn how we should add this `ms_mode=secure` via configMap? If don't set it then CI will still fail.

Reply:
It should be an option in operator.yaml, see [here](https://github.com/rook/rook/blob/release-1.12/deploy/examples/operator.yaml#L489-L491).

Reply:
seems like upgrade and helm test are still failing, something is still missing